src/Form/SitepackageType.php

```diff
@@ -44,34 +44,32 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'Base Package',
                 'choices' => [
                     'Bootstrap Package' => 'bootstrap_package',
-                    'Fluid Styled Content' => 'fluid_styled_content',
+                    'Minimal Site Package' => 'fluid_styled_content',
```

Comment:
It has to be clear that this one is using fluid styled content as rendering engine, i am prepping to move the information out of here to propably also include a description. for now i would like to not adjust it here.

Reply:
Fluid Styled Content is a Core extension and as long as that is the standard way of providing templates for content elements I think we do not have to make this information very clear. It is the current core way of doing things. 

Currently there is no choice ppl have to make here. Either they use fsc or write all the templates for ces themselves / depend on a third party package like the bootstrap package or content blocks.

Reply:
I reverted the change for now let us discuss in a follow up